packages/cli/src/open/open.js

```diff
@@ -37,9 +37,11 @@ async function runCommand(options) {
   if (!representativeLhrs.length) {
     process.stdout.write('No available reports to open. ');
   }
+  
+  const targetUrls = new Set([].concat(options.url || []));
```

Comment:
aw shoot, sorry @colourgarden but typescript doesn't like my too clever suggestion 😅 
```suggestion
  const targetUrls = typeof options.url === 'string' ? 
    [options.url] : 
    (options.url || []);
```